scpsl/README.md

```diff
@@ -0,0 +1,18 @@
+# SCPSL-Pterodactyl-Egg
+
+A Pterodactyl Egg for SCP: Secret Laboratory
+
+This game needs about a minimal of 3096MB of RAM to run smooth.
+
+# How to import this egg?
```

Comment:
This and below isn't really necessary as it's in the main repo and we call it out all over the discord.

Reply:
I changed the readme